index.html

```diff
@@ -5009,6 +5009,36 @@ <h3>Cryptography Suites and Libraries</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Key Management</h3>
+        <p>
+The security of most digital signature algorithms, which are used to secure
+<a>verifiable credentials</a> and <a>verifiable presentations</a>, is dependent
+on the quality and protection of its <em>private signing key</em>. Guidance in
+the management of cryptographic keys is a large subject and the reader is
+referred to [[NIST-SP-800-57-Part-1]] for more extensive recommendations and
+discussion. As strongly recommended in both [[FIPS-186-5]] and
+[[NIST-SP-800-57-Part-1]], a private signing key is not to be re-used for other
+purposes, such as encryption and signing.
+        </p>
+        <p>
+Private signing keys and <em>public verification keys</em> are strongly
+advised to have limited <em>cryptoperiods</em> [[NIST-SP-800-57-Part-1]], where
+a <em>cryptoperiod</em> is &quot;the time span during which a specific key is
+authorized for use by legitimate entities or the keys for a given system will
+remain in effect.&quot; [[NIST-SP-800-57-Part-1]] gives extensive
+guidance on cryptoperiods for different key types under different situations
+and generally recommends a 1-3 year cryptoperiod for a private signing key.
```

Comment:
The keys are not advised; their administrators/controllers are.
```suggestion
[[NIST-SP-800-57-Part-1]] strongly advises that private signing keys and
<em>public verification keys</em> have limited <em>cryptoperiods</em>, where
a <em>cryptoperiod</em> is &quot;the time span during which a specific key is
authorized for use by legitimate entities or the keys for a given system will
remain in effect.&quot; [[NIST-SP-800-57-Part-1]] gives extensive
guidance on cryptoperiods for different key types under different situations,
and generally recommends a 1-3 year cryptoperiod for a private signing key.
```